nym-wallet/src/components/NymCard.tsx

```diff
@@ -21,7 +21,11 @@ export const NymCard: React.FC<{
 }> = ({ title, subheader, Action, Icon, noPadding, borderless, children, dataTestid }) => (
   <Card variant="outlined" sx={{ overflow: 'auto', ...(borderless && { border: 'none', dropShadow: 'none' }) }}>
     <CardHeader
-      sx={{ p: 3, color: (theme: Theme) => theme.palette.text.primary }}
+      sx={{
+        p: 3,
+        color: (theme: Theme) => theme.palette.text.primary,
```

Comment:
inside the `sx` prop you dont need to access the theme directly. You can use a shorthand `text.primary`. Try it. It should work.